README.md

```diff
@@ -56,9 +56,10 @@ MUX_TOKEN_ID=[YOUR_TOKEN_ID]
 MUX_TOKEN_SECRET=[YOUR_TOKEN_SECRET]
 ```
 
-### OPTIONAL Manual Setup
+### Manual Setup
 
-If you choose to do any of the init steps manually.
+<details>
+<summary><strong>Click to see the manual init steps.</strong></summary>
```

Comment:
I did not know this was a thing...cool. I assume it'll pass through in our docs parsing of the readme?